package.json

```diff
@@ -63,6 +63,7 @@
     "classnames": "2.2.1",
     "click-outside": "^2.0.2",
     "clipboard": "^1.7.1",
+    "commonmark": "^0.28.1",
```

Comment:
Why `commonmark`?

Reply:
Also, do you think we should pin the version here?

Reply:
> Why `commonmark`?

I thought using a Markdown parser library made more sense than doing it myself: that could be easily a different project since it isn't something trivial to develop.
I used a commonmark implementation for Elixir in the past, I knew it was fast and reliable. Also, I asked in Slack's #javascript channel and was one of the recommendations. This javascript implementation project is active and well documented, so it was my first choice. Now I've found it lacks one feature and I've switched to markdown-it (which is another implementation of the commonmark specification) in my latest work: the parser does not store the delimiters used in the source, which I use for the live preview part of the project.

> Also, do you think we should pin the version here?

I totally missed this one. I usually use npm to manage my node dependencies, and I had the `npm config set save-prefix='~'` enabled in my local npm config since a long time ago. Yarn doesn't seem to use this setting, and it used the `^` to save the package. I like to use `~` because that way only the patch part is updated, which means any bugs would be fixed in further updates. But maybe you have another policy in Automattic and you're more conservative in this aspect (which I totally understand) and you always use fixed versions of packages. Anyway, thanks for pointing this out!